src/frontend/planner_test/tests/testdata/predicate_pushdown.yaml

```diff
@@ -300,8 +302,10 @@
     create table t1(v1 timestamp with time zone, v2 int);
     select * from t1 where v1 > now() + '30 min' and v2 > 5;
   optimized_logical_plan: |
-    LogicalFilter { predicate: (t1.v1 > (Now + '00:30:00':Interval)) }
-    └─LogicalScan { table: t1, columns: [t1.v1, t1.v2], predicate: (t1.v2 > 5:Int32) }
+    LogicalJoin { type: LeftSemi, on: (t1.v1 > $expr1), output: all }
```

Comment:
hmm, it's weird that the stream optimization is applied to optimized_logical_plan...?

Reply:
Sorry, I have changed the `optimized_logical_plan` default meaning from the batch to the streaming before. I will change it back to the batch logical optimization.

Reply:
can we remove them and just maintain the `batch_plan` and `stream_plan`? or support `optimized_logical_plan_for_batch` and `optimized_logical_plan_for_stream`